resolwe_bio/tests/processes/test_enrichment.py

```diff
@@ -21,6 +23,7 @@ def test_go_enrichment_dicty(self):
         enrichment = self.run_process('goenrichment', inputs)
         self.assertJSON(enrichment, enrichment.output['terms'], '', 'go_enriched_terms_dicty.json.gz')
 
+    @with_resolwe_api_host
```

Comment:
Why don't you add this decorator to the `setUp` method, so all tests can use it?